webrender_traits/src/display_item.rs

```diff
@@ -129,17 +129,16 @@ impl ComplexClipRegion {
     }
 
     //TODO: move to `util` module?
-    /// Return a maximum aligned rectangle that is fully inside the clip region.
+    /// Return an aligned rectangle that is fully inside the clip region.
```

Comment:
Makes sense, yeah. I was just shocked by the removal of my carefully computed mathematical constant that was supposed to optimize everything :)